app/controllers/users_controller.rb

```diff
@@ -0,0 +1,36 @@
+class UsersController<ApplicationController
+  def new
+    @user = User.new
```

Comment:
Passed in this instance variable so that if the form is not properly filled out that we can re render the form with populated fields the user filled in